Cocktails9/Cocktails9/Login.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  login.swift
+//  Cocktails9
+//
+//  Created by Mateja Mitic on 26.11.24..
+//
+
+import SwiftUI
+
+struct Login: View {
+    
+    @State private var username: String = ""
+    @State private var password: String = ""
+    @State private var isLoggedIn: Bool = false
```

Comment:
you don't need this Bool. I see it's only used in one `func` down below, so it can be that `func`'s local `var` (or `let`)